README.md

```diff
@@ -87,7 +87,10 @@ The license is Apache 2.0, see LICENSE-2.0.txt.
 
 ### Binary Releases
 
-You can find published releases (compiled for Java 6 and above) on
+Version 1.2.1 and earlier were built for Java 6, while newer
+versions will be built for Java 8.
```

Comment:
Do you know what you intend to use for the version number for the release that this patch goes into?  Seems like it would be fairly drastic to jump from Java 6 to Java 8 in a 1.2.2?